web/src/index.jsx

```diff
@@ -34,6 +36,11 @@ console.log(
 // Plug in the sagas
 sagaMiddleware.run(sagas);
 
+// Switches Main component if historical view feature google is used
+const urlSearchParams = new URLSearchParams(window.location.search);
+const isHistoryFeatureEnabled = Object.fromEntries(urlSearchParams.entries())?.feature === 'history';
```

Comment:
I assume we need this because it's hasn't booted up the feature flag yet. I am wondering if there is a way for us to have a central feature flag function that always works?

Or at least sort of similar to how translations work where we have one function to be used inside of react components and one outside if that makes sense.

Reply:
It's because we can't use the hook outside of a component. But yeah, I guess the hook could actually use import this functionality from somewhere..
I'd prefer to address this next time we need it though :)

Reply:
The hook is relying on [useLocation from react-router](https://v5.reactrouter.com/web/api/Hooks/uselocation), so rewriting this will actually be a bit tricky I think..